packages/calypso-e2e/src/lib/base-block.ts

```diff
@@ -0,0 +1,20 @@
+import { ElementHandle } from 'playwright';
+
+/**
+ * Base class for the Block type objects.
+ */
+export class BaseBlock {
+	[ x: string ]: any;
+	static block: ElementHandle;
```

Comment:
If we do keep this base class, I think we don't want this to be static, right? This should be unique to each instance of this class?

Reply:
Ah, really good catch - thanks! This would have had an unintentional side effect when more than 1 block of the same type was to be inserted into the same editor, I assume.